src/CommunityToolkit.Maui.MediaElement/MediaElement.shared.cs

```diff
@@ -377,6 +389,24 @@ public void Stop()
 		Handler?.Invoke(nameof(StopRequested));
 	}
 
+	/// <summary>
+	/// Restores screen position.
+	/// </summary>
+	public void RestoreScreen()
+	{
+		OnRestoreScreenRequested();
+		Handler?.Invoke(nameof(RestoreScreenRequested));
+	}
+
+	/// <summary>
+	/// Sets full screen.
+	/// </summary>
+	public void FullScreen()
```

Comment:
Let's ensure that we are using verbs for method names, eg `EnlargeVideoToFullScreen` or something similar that demonstrates the action that the method will perform

Reply:
Method name was switched from ``` FullScreen() to EnlargeVideoToFullScreen()```